debug/examples/shop.json5

```diff
@@ -73,13 +75,19 @@
     },
     columns: 1,
     assetKeys: {
-        background: "background",
+        // background: "background", // if a string is passed, that asset is used for all panes
+        background: {
+            menu: "", // leave blank for a transparent BG
```

Comment:
if you don't provide a `background.menu` would it error or produce a transparent BG too?

Reply:
Omitting the key entirely produces a transparent BG- I've updated the comment for clarity.

Reply:
a9c902c